contract/models/contract.py

```diff
@@ -112,6 +112,20 @@ class ContractContract(models.Model):
         inverse_name="contract_id",
         string="Modifications",
     )
+    backend_view_id = fields.Integer(
+        store=False, compute="_compute_backend_view_id", string="Backend view"
+    )
+
+    def _compute_backend_view_id(self):
```

Comment:
This is not the idea I had in mind. There's a standard method that returns the view to obtain. Can't this be used?